app/frontend/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,49 @@
+// External Imports
+import React, { Fragment } from "react";
+import { useNavigate } from 'react-router-dom';
+
+// Internal Imports
+import { HeaderNav } from "components/components";
+import { Button } from "components/components";
+import { logoHorizontal, logoStacked, notFoundPageImg } from "assets/images/images";
+
+function NotFoundPage() {
+  const navigate = useNavigate();
+  return (
+    <Fragment>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier/1" },
+          { name: "Projects", link: "/demo" },
+        ]}
+      />
+      <main>
+        <div className="not-found-container flex-container justify-between align-center gap-3">
```

Comment:
Out of curiosity, what is `gap-3` doing here?

Reply:
The `gap-3` is a silly way to make some space between the divs on mobile view

Perhaps I got too excited trying implement the new utility @bzzz-coding created :stuck_out_tongue: 

<details><summary>See screenshot</summary>

![image](https://user-images.githubusercontent.com/73561520/232247688-316ec984-0dcd-4dc7-b009-125095723197.png)

</details>